racket/collects/pkg/private/new.rkt

```diff
@@ -62,20 +62,51 @@
     (make-directory name)
     (parameterize ([current-directory name])
 
-      ;; LICENSE.txt
-      (with-output-to-file "LICENSE.txt"
+      ;; LICENSE files
+      (displayln "Generating LICENSE-APACHE and LICENSE-MIT files. You are free to change the license.")
```

Comment:
Is there any reason these don't use the `.txt` file extension?

Reply:
Hi @benn

@sorawee asked @samth and Sam said no to having a txt extension.
 
See https://github.com/racket/racket/issues/1570#issuecomment-501449180

Kind regards, 
Stephen